src/templates/CoveringLetter/types.ts

```diff
@@ -14,3 +13,9 @@ export interface CoveringLetter extends v2.OpenAttestationDocument {
     };
   };
 }
+
+export type CoveringLetterSchemaV2 = v2.OpenAttestationDocument & CoveringLetter;
+export type CoveringLetterSchemaV3 = v3.OpenAttestationDocument & {
+  credentialSubject: CoveringLetter;
```

Comment:
Ohh... So the so-called covering letter is actually just the `credentialSubject` of the whole document. Something new to me.😂

Reply:
that's assuming all data related should be in `credentialSubject` as per vc specs, some other [examples](https://w3c-ccg.github.io/vc-examples/). in anyways, for as long there is a change in data schema, renderer end must have a new template to handle / render it.